web/sites/default/config/webform.webform._template_standard_webform.yml

```diff
@@ -0,0 +1,205 @@
tion_display': before\n    '#options': disabilities\n    '#states':\n      visible:\n        ':input[name=\"demographic_disability\"]':\n          value: 'Yes'\n      required:\n        ':input[name=\"demographic_disability\"]':\n          value: 'Yes'\n  section_demographic_tribal:\n    '#type': webform_section\n    '#title': 'American Indian/Alaska Native Tribal Affiliation Category'\n    '#description': '<p>Are you an enrolled member and/or a descendant of a Federal or State recognized American Indian Tribe or Alaskan Native Village/Corporation? Mark and describe all that applies.</p>'\n    '#title_tag': h3\n    demographic_tribal_enrolled_member:\n      '#type': checkbox\n      '#title': 'Enrolled Member'\n    demographic_tribal_descendant:\n      '#type': checkbox\n      '#title': Descendant\n    demographic_tribal_affiliation:\n      '#type': textfield\n      '#title': 'Tribal Affiliation(s)'\n      '#states':\n        visible:\n          - ':input[name=\"demographic_tribal_enrolled_member\"]':\n              checked: true\n          - or\n          - ':input[name=\"demographic_tribal_descendant\"]':\n              checked: true\n        required:\n          ':input[name=\"demographic_tribal_enrolled_member\"]':\n            checked: true\n          ':input[name=\"demographic_tribal_descendant\"]':\n            checked: true\nsection_contact_information:\n  '#type': webform_section\n  '#title': 'Contact Information'\n  contact_name:\n    '#type': textfield\n    '#title': 'Your Name'\n  contact_email:\n    '#type': textfield\n    '#title': 'Your Email'\n    '#description': 'We will use your email to respond to your feedback or request.'\n    '#input_mask': '''alias'': ''email'''\n    '#required': true\n  contact_phone:\n    '#type': textfield\n    '#title': 'Your Phone'\n    '#description': 'Please provide your phone number in case we need additional information.'\n    '#input_mask': '(999) 999-9999'\n  contact_address:\n    '#type': webform_address\n    '#title': 'Your Mailing Address'\n    '#title_display': before\n    '#default_value':\n      address: ''\n      address_2: ''\n      city: ''\n      state_province: ''\n      postal_code: ''\n      country: 'United States'\n    '#address__title': Addrerss\n    '#address_2__title': 'Address Line 2'\n    '#address_2__title_display': invisible\nsection_business:\n  '#type': webform_section\n  '#title': 'Business Information'\n  business_name:\n    '#type': textfield\n    '#title': 'Business or Organization Name'\n  business_phone_number:\n    '#type': textfield\n    '#title': 'Business Phone Number'\n    '#input_mask': '(999) 999-9999'\n  business_address:\n    '#type': webform_address\n    '#title': 'Business Address'\n    '#address__title': 'Street Address'\n    '#address_2__title_display': invisible\n    '#state_province__title': State\n    '#postal_code__title': 'ZIP Code'\n    '#country__access': false\ninternal_channel:\n  '#type': hidden\n  '#title': channel\n  '#default_value': webform"
```

Comment:
Need to correct misspelling of "Addrerss"

Reply:
Fixed.